packages/entities/entities-certificates/src/components/CACertificateList.vue

```diff
@@ -13,6 +13,7 @@
       preferences-storage-key="kong-ui-entities-ca-certificates-list"
       :query="filterQuery"
       :table-headers="tableHeaders"
+      :use-action-outside="useActionOutside"
```

Comment:
@mptap Should this line be removed? Because `EntityBaseTable` does not have the `use-action-outside` prop.